README.md

```diff
@@ -1,2 +1,62 @@
 # docker-pdf-signer
 A lightweight Dockerized CLI tool that adds an SVG signature to a PDF without local installations. Mount a folder, specify page, position, and scale, and the container embeds the signature as true vector graphics. Outputs a signed PDF with deterministic placement and full multi-page support.
+
+## Usage
+
+### Using the Pre-built Docker Image from Docker Hub
+
+The easiest way to use docker-pdf-signer is to pull the pre-built image from Docker Hub:
+
+```bash
+docker pull jasdefer/docker-pdf-signer:latest
+```
+
+Then run the container with your PDF and signature files:
+
+```bash
+docker run --rm -v $(pwd):/work jasdefer/docker-pdf-signer:latest \
+  --pdf test.pdf \
+  --signature signature.svg \
+  --page 1 \
+  --rel-x 0.5 \
+  --rel-y 0.1 \
+  --scale 0.3
+```
+
+### Building Locally
+
+If you prefer to build the Docker image yourself:
+
+```bash
+docker build -t docker-pdf-signer .
+docker run --rm -v $(pwd):/work docker-pdf-signer \
```

Comment:
This is a bit redundant. Only show that the image can be pulled directly from docker hub or alternative be build locally. Afterwards present the run command.